packages/common/src/Adaptor.js

```diff
@@ -431,7 +402,7 @@ export function fields(...fields) {
 export function merge(dataSource, fields) {
   return state => {
     const initialData = source(dataSource)(state);
-    const additionalData = expandReferences(fields)(state);
+    const [additionalData] = newExpandReferences(state, fields);
```

Comment:
Here in common, `newExpandReferences` needs to be renamed to `expandReferences`.